### PR TITLE
PLNSRVCE-384: set network isolation label to build task

### PR DESCRIPTION
**Pull request by tkdchen**
Please see the commit message for details of these changes.

### CODE REVIEW

**Comment by tkdchen** [comment]
```bash
$ tkn pr describe -o json nodejs-builder-run-zjl7k | jq '.metadata.labels'
{
  "pipeline-service.io/network-isolation": "true",
  "pipelines.openshift.io/runtime": "nodejs",
  "pipelines.openshift.io/strategy": "s2i",
  "pipelines.openshift.io/used-by": "build-cloud",
  "tekton.dev/pipeline": "nodejs-builder"
}

$ tkn pr describe -o json docker-build-run-hk8n5 | jq '.metadata.labels'
{
  "pipeline-service.io/network-isolation": "true",
  "pipelines.openshift.io/runtime": "generic",
  "pipelines.openshift.io/strategy": "docker",
  "pipelines.openshift.io/used-by": "build-cloud",
  "tekton.dev/pipeline": "docker-build"
}
$ tkn pr describe -o json java-builder-run-zj69l | jq '.metadata.labels'
{
  "pipelines.openshift.io/runtime": "java",
  "pipelines.openshift.io/strategy": "s2i",
  "pipelines.openshift.io/used-by": "build-cloud",
  "tekton.dev/pipeline": "java-builder"
}

```

File: pipelines/base/docker-build/patch.yaml [hunk 1]
```diff
@@ -8,6 +8,7 @@
     "pipelines.openshift.io/used-by": "build-cloud"
     "pipelines.openshift.io/runtime": "generic"
     "pipelines.openshift.io/strategy": "docker"
+    "pipeline-service.io/network-isolation": "true"
```
Comment (Michkov) [inline pipelines/base/docker-build/patch.yaml:11]: this should be applied only on hacbs pipelines

**Comment by lcarva** [comment]
Maybe using the native (experimental) hermetic functionality in Tekton Pipelines is a better approach? https://tekton.dev/docs/pipelines/hermetic/

**Pull request closed without merging**